src/components/InputProblem.tsx

```diff
@@ -121,19 +122,17 @@ export const InputProblem = ({
     return trimmedInput.toLowerCase() === trimmedAnswer.toLowerCase()
   }
 
-  const setPersistedState = async (): Promise<void> => {
-    try {
-      if (contentId === undefined || persistor === undefined) {
-        return
-      }
-
-      if (response !== '' || inputDisabled || retriesAllowed !== 0) {
-        const newPersistedData: PersistorData = { userResponse: response, inputDisabled, retriesAllowed, feedback }
-        await persistor.put(contentId, JSON.stringify(newPersistedData))
-      }
-    } catch (err) {
-      console.error(err)
+  const handleFeedback = (userResponse: string, userAttempts: number): string => {
```

Comment:
Just a thought: Right now we end up calling `evaluateInput` multiple times on a submit. The function is pretty light except for `math` problems where we invoke `ce` / `kas`, but I wanted to note it in case there's an alternative implementation that might avoid that :man_shrugging: .

Reply:
Initial thought was we want to compare the the `userResponse` with the `solution` and `evaluateInput` handles the various `comparator` types so I opted for `evaluateInput`. 

I'll give this one some thought to see if there is an alternative.

Reply:
Let's punt on this and revisit in a future "real" implementation.